conf_linux_public.py

```diff
@@ -89,19 +105,15 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
     else:
         return f'{enable_devtoolset} && {command}' #enable new compiler on CentOS
 
-
-PRODUCT_REPOS = [
-    {'name': 'MediaSDK'},
-    # Give possibility to build linux for changes from product configs repository
-    # This repo not needed for build and added only to support CI process
-    {'name': 'product-configs'}
-    #{'name': 'flow_test'},
-]
-
-ENABLE_DEVTOOLSET = 'source /opt/rh/devtoolset-6/enable'
-GCC_LATEST = '8.2.0'
-options["STRIP_BINARIES"] = True
-MEDIA_SDK_REPO_DIR = options.get('REPOS_DIR') / PRODUCT_REPOS[0]['name']
+def check_lib_size(lib, lib_name):
```

Comment:
"lib" should be lib_path or path_to_lib.
lib_name should be removed (also the is no help on lib_name).